### DISCUSSION TITLE
Keep tar.xz installer?

### DISCUSSION THREAD

**Discussion by jwhite007** [pyenv/pyenv#2180] [Q&A] [answered]
Is there a way to configure pyenv to keep the tar.xz installers in a cache somewhere rather than them being deleted from the temp directory after installation, an environmental variable or such?

**Comment by joshfriend** [comment, marked as answer]
From the README:

> `PYTHON_BUILD_CACHE_PATH`, if set, specifies a directory to use for caching downloaded package files.

Does this answer your question?

**Reply by jwhite007** [reply]
@joshfriend
Thank you. I searched the README at https://github.com/pyenv/pyenv for "cache" and came up with nothing. To which README are you referring?
